apps/omg_watcher/lib/block_getter/core.ex

```diff
@@ -558,6 +566,13 @@ defmodule OMG.Watcher.BlockGetter.Core do
         end
 
       nil ->
+        _ =
+          Logger.warn(fn ->
```

Comment:
I don't think that should be a warning. I think we can have 3 ethereum blocks without any plasma block published on them.

Reply:
this isn't the case. We're looking up a particular plasma block number in submissions - `nil` case is when you know a particular plasma block, but you can't find it's exact submission (because we're limiting ourselves to 2000 eth blocks surrounding the persisted `synced_height`). In such case, everything might work fine (if `synced_height` is correct), but might not, and then the warning can lead one to where the problem is.